### ISSUE TITLE
accuracy of your SegNet model

### ISSUE THREAD

**Issue by hurricane2018** [nshaud/DeepNetsForEO#38]
@nshaud  
I just run the default PyTorch code SegNet_PyTorch_v2.ipynb. However, the accuracy of Vaihingen is about 86%,  not as good as your paper. Then I read your paper again, and I found that you have made some change. Do you have any plan to share the code in the BeyondRGB paper?

**Comment by nshaud** [comment]
Thanks for your interest. The code from the BeyondRGB paper uses multiple data sources (DSM + RGB), it is a multimodal network. I do not have the time right now to update the PyTorch code with this model but I hope to do it in the near future.

**Comment by hurricane2018** [comment]
Yes. I know in your paper you use DSM+RGB. However, in the following table, you achieve about 89.4 and 90% on each dataset, when the SegNet was used in your experiment. When I use the Pytorch code in your repository, it only gets about 86%. 

![image](https://user-images.githubusercontent.com/36166137/60069501-b19e7400-9745-11e9-95b4-2e1e6cc2bc74.png)

![image](https://user-images.githubusercontent.com/36166137/60069513-b9f6af00-9745-11e9-9574-5f88be4ac992.png)

**Comment by hurricane2018** [comment]
In addition, what are your training dataset and validation dataset in Vaihingen and Potsdam? You did not explain it in your article. 

Are these your training dataset and testing dataset in your article?

segnet_vaihingen_128x128_fold1_iter_60000.caffemodel (112.4 Mo) (backup link): pre-trained model on the ISPRS Vaihingen dataset (trained on tiles 1, 3, 5, 7, 11, 13, 15, 17, 21, 23, 26, 28, 30, validated on tiles 32, 34, 37).
potsdam_rgb_128_fold1_iter_80000.caffemodel (112.4 Mo) (backup link) : pre-trained model on the ISPRS Potsdam dataset (RGB tiles, trained on (3, 12), (6, 8), (4, 11), (3, 10), (7, 9), (4, 10), (6, 10), (7, 7), (5, 10), (7, 11), (2, 12), (6, 9), (5, 11), (6, 12), (7, 8), (2, 10), (6, 7), (6, 11), validated on tile (2, 11), (7, 12), (3, 11), (5, 12), (7, 10), (4, 12)).

Best,

**Comment by nshaud** [comment]
IIRC final results reported on the article are trained on the whole training set and metrics are computed using the ISPRS official test set. Different test splits can have different metrics.
